digital-deck-ui/src/store/index.js

```diff
@@ -8,8 +8,8 @@ export default createStore({
     state: {
         sessionId: -1,
         playerId: -1,
-        playerCards: null,
-        tableCards: null,
+        playerCards: [],
```

Comment:
I set these to null so the tests can check whether `initSession` actually grabbed the data. Do you need these to be empty arrays for any reason?

Reply:
The previous method I was working with showed some weird behavior with them being null. I changed my approach later but forgot to change these later.